apps/comps/e2e/tests/spot-live-competition.test.ts

```diff
@@ -25,15 +27,48 @@ import {
 } from "@recallnet/test-utils";
 import type { AdminAgentResponse } from "@recallnet/test-utils";
 
-import { db } from "@/database/db.js";
-import { ServiceRegistry } from "@/services/index.js";
+import { db } from "@/lib/db";
+import { createLogger } from "@/lib/logger";
+import {
+  agentRepository,
+  balanceRepository,
+  competitionRepository,
+  spotLiveRepository,
+  tradeRepository,
+} from "@/lib/repositories";
+import {
+  portfolioSnapshotterService,
+  priceTrackerService,
+} from "@/lib/services";
 
 describe("Spot Live Competition", () => {
   let adminApiKey: string;
 
+  // Create test-specific mock RPC provider for spot live tests
```

Comment:
API app sets up this mock always, here in the comps app I'm setting it up only in this test